cmd/buildkit-nix/frontend.go

```diff
@@ -55,6 +57,7 @@ func frontendBuild(nixImage string) client.BuildFunc {
 		localDfSt := llb.Local(localNameDockerfile,
 			llb.SessionID(c.BuildOpts().SessionID),
 			dockerfile2llb.WithInternalName("local dockerfile"),
+			llb.FollowPaths([]string{dfName}),
```

Comment:
this is actually going to get complicated.. relative paths from the Dockerfile won't work anymore since this is being copied to `/dockerfile/{dfName}` and the context is in `/context`